packages/payment-processor/src/contracts/MultisigContract.ts

```diff
@@ -0,0 +1,420 @@
+/* tslint:disable */
+/* eslint-disable spellcheck/spell-checker */
+/* Generated by ts-generator ver. 0.0.8 */
+
+import { Contract, ContractTransaction, EventFilter, Signer } from 'ethers';
+import { Listener, Provider } from 'ethers/providers';
+import { Arrayish, BigNumber, BigNumberish, Interface } from 'ethers/utils';
+import { TransactionOverrides, TypedEventDescription, TypedFunctionDescription } from '.';
+
+interface MultisigContractInterface extends Interface {
```

Comment:
Should be rename in something more specific like: `GnosisSafeMultisigContractInterface` ou `IGnosisSafeMultisigContract`